components/toast/toast.jsx

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import UnreadBelowIcon from 'components/widgets/icons/unread_below_icon';
+import CloseIcon from 'components/widgets/icons/close_icon';
+
+const MIN_TOAST_HEIGHT = 1000;
+
+export default class Toast extends React.PureComponent {
+    static propTypes = {
+        onClick: PropTypes.func.isRequired, // required?
```

Comment:
Yup, I think this can be required or we need to be do a defensive to check for the value of onClick before calling `this.props.onClick()`